src/components/Contact/ContactForm.module.css

```diff
@@ -0,0 +1,10 @@
+body{
```

Comment:
We should avoid this tag specific css as they will bleed

Reply:
Moved styling to the parent page and used a specific class to style background.